css/home-page.css

```diff
@@ -0,0 +1,171 @@
+.home-page{
+    background-color: lightgoldenrodyellow;
+}
+
+.home-header{
+    align-items: center;
+    background-color: gray;
+    background-image: url(background-soft2.jpeg);
+    background-repeat: no-repeat;
+    border: 2px solid black;
+    border-radius: 5px;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap; 
```

Comment:
fixed

Reply:
No you didn't...

Reply:
sorry i don't get it, what unnecessary?

Reply:
now i'm more confused than ever

Reply:
Ok, the "flex-wrap: wrap;" is indeed necessary. There were also other problems that let me assume it wasn'tnecessary.
Next time let me notice that it is required in case I'm wrong.
Since you wrote "fixed" I thought you wrongly wrote "it's necessary".
For that reason I answered "no you didn't" because there were no changes in the code.